src/main/java/Duke.java

```diff
@@ -1,10 +1,208 @@
+    private static final ArrayList<Task> listTasks= new ArrayList<>(1);
+    public static int jobCount = 0;
+
+    /**
+     * Function to print Logo and Hello
+     */
+    static void printHello() {
+        String logo = "   _____  .__        \n"
+                + "  /  _  \\ |__|______ \n"
+                + " /  /_\\  \\|  \\_  __ \\\n"
+                + "/    |    \\  ||  | \\/\n"
+                + "\\____|__  /__||__|   \n"
+                + "        \\/           \n";
+        System.out.println("\tHello from\n" + logo + "\n");
+        System.out.println("____________________________________________________________\n");
+        System.out.println("\tSup! I'm Air\n" + "\tHow can I help you out today?\n");
+        System.out.println("____________________________________________________________\n");
+    }
+    private static void writeToFile(String filePath, String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filePath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+    /**
+     * Function to print the list of tasks on typing list
+     */
+    static void printTaskList(int index) {
+        for (int i = 0; i < index; i++) {
+            System.out.println((i + 1) + ".  " + listTasks.get(i).printDetails());
+        }
+    }
+
+    /**
+     * Function to create task and add it to the listTasks
+     */
+
+    static void newTask(Task userTask) {
+        jobCount++;
+        listTasks.add(userTask);
+    }
+
+    private static void gibberishError() throws NoSuchCommandException {
+        throw new NoSuchCommandException();
+    }
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+        Read obj= new Read("./data");
+        listTasks.addAll(obj.importedTasks);
+        jobCount=obj.existingTaskCount;
+        String inputCommand;
+        printHello();
+        Scanner scanIn = new Scanner(System.in);
+        /** while loop works only till the word bye isn't typed **/
+        while (!(inputCommand = scanIn.nextLine()).equals("bye")) {
+//            String inputCommand = scanIn.nextLine();
+            String[] splitInputs = inputCommand.split(" ", 2);
+            if (inputCommand.equals("list")) {
+                printTaskList(jobCount);
+//                System.out.println(Task.jobCount);
+            } else if (inputCommand.length() >= 4) {
+                /** Above condition used to prevent code for running for nonsensical commands or for commands where no parameters are defined*/
+                if (inputCommand.contains("done")) {
+                    try {
+                        int taskNumber = Integer.parseInt(inputCommand.substring(5, 6));
```

Comment:
I'm not sure if the program may crash if you enter a command but the 2nd parameter is not a number.
Be careful as you program should not crash when the user enters a invalid command